devops/ansible/nginx_install.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install nginx server 
+  hosts: stagingserver
+  remote_user: wimops
+  roles:
+    - nginx
+...
```

Comment:
I thought the story was just the server and the rest would be added later. Today, I realized it should be the entire package as the kb is , so I've been testing out adding the rest (php,mariadb,wordpress). I'll continue to work on that and add the ssl as well.